lib/Application.vala

```diff
@@ -206,65 +206,5 @@ namespace Granite {
                 Granite.Services.Logger.DisplayLevel = Granite.Services.LogLevel.DEBUG;
             }
         }
-
-        /**
-         * This methods creates a new App Menu
-         *
-         * @param menu the menu to create the App Menu for
-         *
-         * @return app_menu
-         */
-        [Version (deprecated = true, deprecated_since = "0.4.2", replacement = "Gtk.MenuButton")]
-        public Granite.Widgets.AppMenu create_appmenu (Gtk.Menu menu) {
-
-            var app_menu = new Granite.Widgets.AppMenu.with_app (this, menu);
-            app_menu.show_about.connect (show_about);
-
-            return app_menu;
-        }
-
-        protected Granite.Widgets.AboutDialog about_dlg;
-
-        /**
-         * This method shows the about dialog of this app.
-         *
-         * @param parent This widget is the window that is calling the about page being created.
-         */
-        [Version (deprecated = true, deprecated_since = "0.4.2", replacement = "")]
-        public virtual void show_about (Gtk.Widget parent) {
```

Comment:
I'm a little worried about removing this because it seems to be used as recently as 2 months ago: https://github.com/mirkobrombin/ElementaryPython

I think it would be better to leave it in but make it either do nothing or throw a https://valadoc.org/gtk+-3.0/Gtk.AboutDialog.html